Formula/etcher-cli.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher.git",
+    :tag => "v1.4.4",
+    :revision => "cbd531e161a12721fb763b526781f66ef8237ac9"
+
+  depends_on "python" => :build
+  depends_on "jq"
+  depends_on "node"
+
+  def install
+    Language::Node.setup_npm_environment
+    system "make", "RELEASE_TYPE=production", "cli-develop"
+    system "make", "RELEASE_TYPE=production", "package-cli"
+
+    inreplace "Makefile", "	mocha", "	npx -p node@6 mocha"
+    system "make", "test-cli"
+
+    cd "dist/Etcher-cli-1.4.4-darwin-x64" do
+      bin.install "etcher"
+      prefix.install "node_modules"
```

Comment:
I don't think a top level `node_modules` in `prefix` is the perfect location for this, butI know it is more or less the required location for your current setup. It would be better to use `Language::Node.std_npm_install_args` here.

Also you are unnecessarily installing a lot of electron related dependencies here, which could explain your long build times (2 min for me for the cli only build).